Source/Extensions/UIView+Stacking.swift

```diff
@@ -62,6 +62,12 @@ extension UIView {
     }
 }
 
+extension CGSize {
+    static public func equalEdge(_ size: CGFloat) -> CGSize {
```

Comment:
You should rename this to equalEdge(_ edge: CGFloat).  "size" means a two dimensional W x H object in Swift whereas here it represents a single dimension.

Reply:
I change it.